src/visualization/types/SimpleTable/style.scss

```diff
@@ -57,8 +55,5 @@
   margin-right: $cf-marg-b;
   color: $g13-mist;
   font-weight: $cf-font-weight--medium;
-}
-
-.visualization--simple-table--paging-button {
-  margin-left: $cf-marg-b;
+  width: 189px;
```

Comment:
padding-bottom: $cf-marg-b

<3

I don't know if that'll do what I'm hoping it'll do but worth a shot, just to give those dapper scroll bars some breathing room